audio/wave_overrides.inc

```diff
@@ -0,0 +1,10 @@
+WaveOverrides:
+	dw .override
+	dw .override
+	dw .override
+	dw .override
+	dw .override
+	dw .override
+.override
```

Comment:
Add blank lines between chunks of code like this; e.g. for a pointer table with pointers to various data pieces defined below it, the table and each piece are separated by blanks.